.changeset/delicate-dark-coin.md

```diff
@@ -0,0 +1,7 @@
+---
+"@utrecht/{naam-component}-css": minor
+---
+
+- Added metadata for link-social tokens.
+- Added missing token for --utrecht-link-social-icon-size variable which was found in the mixin.
+- Changed order of tokens so they are easier to scan.
```

Comment:
```suggestion
```

Deze entry is handig als commit message intern, maar niet nodig voor de gebruikers van de component.